### PR TITLE
Sdk 1.x

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
conversion to operator sdk 1.x
also includes:

- better management of status and condition
- uninterruptible context propagation for better linux signal reaction
- fix on edge case when comparing patches
- helm chart with best practices

### CODE REVIEW

File: .github/workflows/push.yaml [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+      - name: create and copy bundle to community operators
+        shell: bash
+        run: | 
+          make bundle IMG=quay.io/${{ github.repository_owner }}/$(basename $GITHUB_REPOSITORY):v${VERSION} VERSION=${VERSION} DEFAULT_CHANNEL=alpha
+          sed  's/bundle\///g' bundle.Dockerfile > bundle/Dockerfile
+          sed -i '/replaces: '"$(basename $GITHUB_REPOSITORY)"'/d' ./bundle/manifests/$(basename $GITHUB_REPOSITORY).clusterserviceversion.yaml
+          mkdir -p ./tmp/community-operators/community-operators/$(basename $GITHUB_REPOSITORY)/${VERSION}
+          /bin/cp -v -R ./bundle/* ./tmp/community-operators/community-operators/$(basename $GITHUB_REPOSITORY)/${VERSION}         
+
+      # - name: Create Pull Request
```
Comment (sabre1041) [inline .github/workflows/push.yaml:245]: Can this section be removed?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
One small request

File: .github/workflows/push.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply .github/workflows/push.yaml:245]: I don't understand you want to remove line 242? for what reason?

Comment (sabre1041) [reply .github/workflows/push.yaml:245]: no. starting at 245. the entire commented out section

Comment (raffaelespazzoli) [reply .github/workflows/push.yaml]: fixed. It needed to be un-commeted.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**